en/plugins/salesforce.md

```diff
@@ -29,6 +29,47 @@ Insert the keys to the Mautic Salesforce plugin and authorize it.
 
 Configure the [field mapping](./../plugins/field_mapping.html).
 
+###Features
```

Comment:
I think there must be a space between the hashes and the heading title to be rendered as the heading:

###Heading
vs
### Heading